Gemfile

```diff
@@ -4,7 +4,13 @@ gemspec
 
 gem "activesupport", "~> 3.0"
 gem "i18n"
-gem "json"
+
+if RUBY_VERSION < "2.0"
```

Comment:
It looks like https://rubygems.org/gems/json says REQUIRED RUBY VERSION: ~> 2.0

I wonder why that was ignored.